index.html

```diff
@@ -382,10 +382,9 @@ <h2>The <dfn>cancelIdleCallback()</dfn> method</h2>
     </section>
     <section data-dfn-for="IdleDeadline">
       <h2>The <dfn>IdleDeadline</dfn> interface</h2>
-      <p>Each <a>IdleDeadline</a> has an associated <dfn>time</dfn>
-      which holds a {{DOMHighResTimeStamp}} representing the absolute time in
-      milliseconds of the deadline. This MUST be populated when the
-      <a>IdleDeadline</a> is created.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <dfn>get deadline time</dfn>, an algorithm that
+      returning a {{DOMHighResTimeStamp}} representing the absolute time in
+      milliseconds of the deadline, initially set to return zero.
```

Comment:
How about?: 

```suggestion
      <p>Each {{IdleDeadline}} has an associated <dfn>get deadline time</dfn> algorithm, which
      returns a {{DOMHighResTimeStamp}} representing the absolute time in
      milliseconds of the deadline. The deadline is initially set to return zero.
```

Random suggestion... but ignore if "get deadline time" is custom algorithm that is passed. 

However, it might be better to just have a "get deadline time" algorithm that takes an {{IdleDeadline}} instance and gives back a DOMHighResTimeStamp. I.e.:

```TS
function getTimeStamp(deadLine: IdleDeadline) => DOMHighResTimeStamp {
    // do stuff with IdleDeadline
    return timestamp.
}
```

Reply:
It is a custom algorithm that is passed from the HTML spec... so I don't think this suggestion is applicable.